op-acceptor/reporting/tree_formatters.go

```diff
@@ -0,0 +1,607 @@
+package reporting
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"strings"
+	"time"
+
+	"github.com/ethereum-optimism/infra/op-acceptor/types"
+	"github.com/ethereum-optimism/infra/op-acceptor/ui"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+// formatDuration formats a duration for display
+func formatDuration(d time.Duration) string {
+	if d < time.Second {
+		return fmt.Sprintf("%dms", d.Milliseconds())
+	}
+	return d.Truncate(time.Millisecond).String()
+}
+
+// TreeHTMLFormatter formats test trees as HTML using the tree structure
+type TreeHTMLFormatter struct {
+	template *template.Template
+}
+
+// NewTreeHTMLFormatter creates a new tree-based HTML formatter
+func NewTreeHTMLFormatter(templateContent string) (*TreeHTMLFormatter, error) {
+	tmpl, err := template.New("tree-report").Funcs(template.FuncMap{
+		"formatDuration": func(d time.Duration) string {
+			if d < time.Second {
+				return fmt.Sprintf("%dms", d.Milliseconds())
+			}
+			return d.Truncate(time.Millisecond).String()
+		},
+		"getStatusClass": func(status types.TestStatus) string {
```

Comment:
those 2 functions (getStatusClass, getStatusText) probably should be consolidated with getStatusString below